src/openforms/submissions/tests/test_tasks_confirmation_emails.py

```diff
@@ -560,6 +560,11 @@ def test_send_confirmation_email_when_appointment_is_changed(self):
         self.assertEqual(message.subject, f"Confirmation mail {_('(updated)')}")
 
     def test_template_is_rendered_in_submission_language(self):
+        """A subset of the components, with particularly weird APIs.
+        Translations on all FormIO components are exercised (until exorcised) in
+        .test_submission_report.DownloadSubmissionReportTests:test_report_is_generated_in_same_language_as_submission
+        """
```

Comment:
```suggestion
        """
        Assert a subset of the components with particularly weird APIs is translated correctly.
        
        Translations on all FormIO components are exercised (until exorcised) in
        ``DownloadSubmissionReportTests.test_report_is_generated_in_same_language_as_submission``
        (in the ``.test_submission_report`` module).
        """
```

Running tests with `-v 2` or higher prints the docstring of the test intead of the test path (following PEP 257 semantics), so make sure to provide test-specific meaningful summaries.